lib/application/constants/images.dart

```diff
@@ -17,6 +17,7 @@ enum ImageKey {
   trendingUpArrow,
 
   // illustrations
+  folderIllustration,
```

Comment:
This is the same as `folder` asset, but since `folder` was exported to fit in the tab, it hasn't enough dimensions to be used in the empty state (it gets extremely low quality if stretched to the desired dimensions)

Reply:
Then maybe `folderBig`?

Reply:
Naming it as "illustration" makes others seem to not adhere to this nomenclature